### PR TITLE
Add the forceRefresh parameter to Installations' getToken method

### PR DESCRIPTION
**Pull request by mmermerkaya**

### CODE REVIEW

File: packages/installations/src/functions/get-token.ts [hunk 1]
```diff
@@ -23,14 +23,17 @@ import { AppConfig } from '../interfaces/app-config';
...
+  forceRefresh?: boolean
+): Promise<string> {
   const appConfig = extractAppConfig(app);
 
   await completeInstallationRegistration(appConfig);
 
   // At this point we either have a Registered Installation in the DB, or we've
   // already thrown an error.
-  return refreshAuthToken(appConfig);
+  return refreshAuthToken(appConfig, forceRefresh);
```
Comment (dwoffinden) [inline packages/installations/src/functions/get-token.ts]: If you don't pass forceRefresh to getToken, this'll be undefined, right? Tbh I'd rather be explicit, make it `= false` here and pass the boolean down? I didn't even realise that that worked with an explicit undefined parameter :/

File: packages/installations/src/functions/get-token.test.ts [hunk 2]
```diff
@@ -226,7 +227,7 @@ describe('getToken', () => {
 
         it('resolves with an auth token', async () => {
           const token = await getToken(app);
-          expect(token).to.equal(AUTH_TOKEN);
+          expect(token).to.be.oneOf([AUTH_TOKEN, NEW_AUTH_TOKEN]);
```
[the remark this answers is not in the document]
Comment (mmermerkaya) [reply packages/installations/src/functions/get-token.test.ts]: It is deterministic, it's just that this test is looping through different situations and making sure that the function always returns a valid auth token, whether it's the one in the DB or it's a new one from the server.

File: packages/installations/src/functions/get-token.ts (hunk 1 above)
Comment (mmermerkaya) [reply packages/installations/src/functions/get-token.ts]: Passing `undefined` explicitly or not passing anything work exactly the same in JS, so I decided to keep the default value definition in the actual function that uses the value, but if it does look that scary, I can change it. :smile:

**Comment by Feiyang1** [comment]
@mmermerkaya  Seems we forgot to update `firebase/index.d.ts`. Can you please update it ?

**Pull request merged**